src/controllers/scripting/legacy/controllerscriptinterfacelegacy.cpp

```diff
@@ -609,6 +609,16 @@ void ControllerScriptInterfaceLegacy::softTakeoverIgnoreNextValue(
     m_st.ignoreNext(pControl);
 }
 
+bool ControllerScriptInterfaceLegacy::softTakeoverIsIgnoring(
+        const QString& group, const QString& name, double parameter) {
+    ControlObject* pControl = ControlObject::getControl(
+            ConfigKey(group, name));
+    if (!pControl) {
+        return false;
+    }
+    return m_st.ignore(pControl, parameter);
```

Comment:
I think you want to implement a a dry run function, right? Ignore() is state full, updates the timestamp and the previous value so that a subsequent set will be never ignored.

Reply:
whoops, yeah, you're right.